api/views.py

```diff
@@ -4,7 +4,7 @@
 from django.conf import settings
 from django.db import transaction
 from django.http import JsonResponse
-from django.core.mail import send_mail
+from django.core.mail import send_mail, EmailMultiAlternatives
```

Comment:
The only way to specify a `reply_to` and an html and text content is to use `EmailMultiAlternatives`. The shorthand `send_mail` does not allow to use a `reply_to`.